monitoring/rid_qualifier/utils.py

```diff
@@ -28,6 +28,9 @@ class EvaluationConfiguration(ImplicitDict):
     min_query_diagonal: float = 100
     """Do not make queries with diagonals smaller than this many meters."""
 
+    reduce_query_variation: bool = True
```

Comment:
Let's make this behavior more self-documenting at the same time as making it more flexible:

```python
repeat_query_rect_period: int = 3
"""If set to a value above zero, reuse the most recent query rectangle/view every this many queries."""
```

and

```python
if (last_rect and
    config.reduce_query_variation > 0 and
    query_counter % config.reduce_query_variation == 0):
```

Reply:
`reduce_query_variation` can now be removed -- since `repeat_query_rect_period=0` doesn't make sense, we can use this as the value to disable the feature.  So, the check can be `config.repeat_query_rect_period > 0` rather than `config.reduce_query_variation > 0` which we currently have but doesn't make sense any way because `config.reduce_query_variation` is a `bool` not an `int`.